client/state/signup/steps/design-type/selectors.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+export function getDesignType( state ) {
+	return get( state, 'signup.steps.designType', '' );
+}
```

Comment:
Selectors belong in `state/selectors` no?

https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#selectors

Reply:
I wasn't sure how inconsistent to get here - all of the selectors related to signup steps are still in the `client/state/signup/steps` location along with the reducer.

Reply:
yeah, I see what you mean. The advantage of the going the `state/selectors` route has to do with caching. https://github.com/Automattic/wp-calypso/blob/master/client/lib/create-selector/index.js#L84

@samouri How strict are we with the adherence to placing all selectors in `state/selectors`? Thanks

Reply:
Hi 👋 !

>Selectors belong in state/selectors no?
Yep!  New selectors should be going there.

> I wasn't sure how inconsistent to get here

The general rule of thumb I've been following is that if there is already a file selecting things out of a specific reducer then I'd rather add more selectors to that file (and refactor later) instead of splitting up the selectors over multiple files/patterns.

In this case in particular it looks like its a brand new file, so my personal rule of thumb would say to place the selector in `state/selectors/index.js`.

> The advantage of the going the state/selectors route has to do with caching.

I don't think thats true.  The main advantage of putting them in state/selectors is more about just having a single place to look for them.  Its just an organizational bit.  You can actually use the `createSelector` function from anywhere Calypso!

Reply:
Sorry @samouri and @psealock  - I jumped in too soon and had merged before you had this conversation.

I'll create another PR to move the selector.